### PR TITLE
add exclusions per scott

### PR DESCRIPTION
**Pull request by mikebell90**

### CODE REVIEW

File: src/main/resources/otj-pmd-rulesets/ot-pmd.xml [hunk 1]
```diff
@@ -71,6 +71,8 @@
         <exclude name="AvoidPrefixingMethodParameters" />
         <exclude name="ShortClassName" />
         <exclude name="GenericsNaming" />
+        <!-- Way too picky @Goofy("star") vs @Goofy(value = "star") is ok -->
+        <exclude name="UnnecessaryAnnotationValueElement" />
```
Comment (adamzr) [inline src/main/resources/otj-pmd-rulesets/ot-pmd.xml]: I like this rule, but agree that it's not worth failing the build over.

Comment (mikebell90) [reply src/main/resources/otj-pmd-rulesets/ot-pmd.xml]: Scott haf tthe issue th at some tool he has auto fills in value=

I like the rule too and like a lot of these... Lcd. You could overide yourself but that's a consistency nightmare

Comment (scottjohnson) [reply src/main/resources/otj-pmd-rulesets/ot-pmd.xml]: I guess the question... why does this rule matter?

Comment (adamzr) [reply src/main/resources/otj-pmd-rulesets/ot-pmd.xml]: doesn't, just stylistic, looks nicer w/o `value`.

**Pull request merged**